Symfony/CS/Fixer/Symfony/AliasFunctionsFixer.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+class AliasFunctionsFixer extends AbstractFixer
+{
+    /** @var string[] stores alias (key) - master (value) functions mapping */
+    private static $aliases = array(
+        'chop' => 'rtrim',
+        'close' => 'closedir',
+        'doubleval' => 'floatval',
+        'fputs' => 'fwrite',
+        'join' => 'implode',
+        'ini_alter' => 'ini_set',
+        'is_double' => 'is_float',
+        'is_integer' => 'is_int',
+        'is_long' => 'is_int',
+        'is_real' => 'is_float',
+        'is_writeable' => 'is_writable',
+        'key_exists' => 'array_key_exists',
+        'magic_quotes_runtime' => 'set_magic_quotes_runtime',
+        'pos' => 'current',
+        'rewind' => 'rewinddir',
```

Comment:
http://php.net/manual/en/function.rewind.php
rewind — Rewind the position of a file pointer

http://php.net/manual/en/function.rewinddir.php
rewinddir — Rewind directory handle


It is not an alias.